azure-iot-device/azure/iot/device/common/mqtt_transport.py

```diff
@@ -80,6 +80,12 @@ class MQTTTransport(object):
     :type on_mqtt_connection_failure_handler: Function
     """
 
+    # Horton uses this variable to make sure we're cleaning up correctly.
+    # This object in particuar is checked because it holds a reference to Paho
+    # and Paho holds references to network resources and we want to be extra sure
+    # that we clean those up correctly.
+    _object_count = 0
```

Comment:
>_object_count [](start = 4, length = 13)

can u include a line saying only for test or perhaps make this variable to be `horton_test_references`  or  `test_object_count` ?